### PR TITLE
Remove xfail marker of ecmp/test_ecmp_sai_value.py

### LINKED ISSUE #11310: test_ecmp_sai_value failed on 202311 image
**Description**
test_ecmp_sai_value failed on 202311 image.
`Failed: sai_hash_seed_config_hash_offset_enable is not set to 1`

https://github.com/sonic-net/sonic-buildimage/pull/17505 
Fix ecmp hash polarization by enable hash seed/offset config on T1 and upgrade xgs SAI to 8.4.35.0.
It needs to be synced into 202311 image.

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

### PR DESCRIPTION
**Pull request by Gfrom2016**
### Description of PR

Summary:
Fixes #11310 
LAG and ECMP hash offset support has merged in 202311 https://github.com/sonic-net/sonic-swss/pull/3138, remove the xfail marker of ecmp/test_ecmp_sai_value.py

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311
- [ ] 202405

#### What is the motivation for this PR?
After https://github.com/sonic-net/sonic-swss/pull/3138, configure hash_offset along with hash_seed attribute is enabled on 202311 onwards on Broadcom T1 platfrom.

#### How did you do it?
Remove the xfail marker of ecmp/test_ecmp_sai_value.py

#### How did you verify/test it?
Run ecmp/test_ecmp_sai_values.py on 7260 T1 DUT and passed.
https://dev.azure.com/mssonic/internal/_build/results?buildId=586225&view=results

### CODE REVIEW

**Comment by ZhaohuiS** [comment]
@Gfrom2016 is the code change included in sonic-buildimage 202405 branch?
please hold on this PR until the new 202311 image is released.

**Comment by Gfrom2016** [comment]
> @Gfrom2016 is the code change included in sonic-buildimage 202405 branch? please hold on this PR until the new 202311 image is released.

I have added 'Request for 202405 image' label, will merge this after the image fix is merged. Thanks for the reminder.

**Comment by wsycqyz** [comment]
PR test re-triggeered.

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Comment by Gfrom2016** [comment]
@StormLiangMS Could you help merge this?

**Review by wangxin — APPROVED** [review]

**Comment by ZhaohuiS** [comment]
@Gfrom2016 if the swss fix is merged in master, we have to remove skip for master and internal.

**Comment by Gfrom2016** [comment]
> @Gfrom2016 if the swss fix is merged in master, we have to remove skip for master and internal.

@ZhaohuiS  sure, I will raise another PR to un-skip the case

**Pull request merged**
